handlers/practitioner_resource_test.go

```diff
@@ -130,6 +130,28 @@ func TestUnitHandleCreatePractitionersResource(t *testing.T) {
 		So(res.Body.String(), ShouldContainSubstring, "last_name is a required field")
 	})
 
+	Convey("Incoming request has invalid last name", t, func() {
+		httpmock.Activate()
+		mockCtrl := gomock.NewController(t)
+		defer httpmock.DeactivateAndReset()
+		defer mockCtrl.Finish()
+
+		body, _ := json.Marshal(models.PractitionerRequest{
+			IPCode:    "1234",
+			FirstName: "F1rst",
+			LastName:  "Last",
+			Address: models.Address{
+				AddressLine1: "addressline1",
+				Locality:     "locality",
+			},
+			Role: constants.FinalLiquidator.String(),
+		})
+		res := serveHandleCreatePractitionersResource(body, mock_dao.NewMockService(mockCtrl), true)
+
+		So(res.Code, ShouldEqual, http.StatusBadRequest)
+		So(res.Body.String(), ShouldContainSubstring, "failed on the 'name_rule' tag")
```

Comment:
I'm not sure if `failed on the 'name_rule' tag` is a descriptive error message. I think we should have something telling the API user what our name rules are, e.g. names shouldn't have any special characters, etc.